pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -287,3 +295,86 @@ func getVolumeGroups(groups []string) ([]vgInfo, error) {
 
 	return vgs, nil
 }
+
+func createNS(r *ndctl.Region, uselimit uint) error {
+	const align uint64 = 1024 * 1024 * 1024
+	realalign := align * r.InterleaveWays()
+	// uselimit is the percentage we can use
+	canUse := uint64(uselimit) * r.Size() / 100
```

Comment:
As you noticed here, `uselimit` is a name that cannot be understood without additional explanation. Why not just call the parameter "percentage"?

Reply:
Renamed the variable.